.vscode/extensions.json

```diff
@@ -7,7 +7,7 @@
 		"dbaeumer.vscode-eslint",
 		"editorconfig.editorconfig",
 		"mikestead.dotenv",
-		"msjsdiag.debugger-for-chrome"
+		"ms-vscode.js-debug"
```

Comment:
I wouldn't update the extensions without discussing it in the channel first, and I would suggest not packaging it as part of this PR - it may delay the PR being merged for various reasons.